pkg/controller/releasemanager/plan/syncApp.go

```diff
@@ -381,3 +383,10 @@ func (p *SyncApp) prometheusRule() *monitoringv1.PrometheusRule {
 		},
 	}
 }
+
+func (p *SyncApp) defaultLabels() map[string]string {
```

Comment:
Was this moved or something? Why's it added here?

Reply:
Yikes this is also cruft thank you.